src/main/java/nextstep/subway/domain/station/StationStatus.java

```diff
@@ -29,4 +29,20 @@ public void distanceValidation(Distance distance) {
     public void splitLineStation(StationPosition stationPosition, Station interStation, Distance distanceToSubtract) {
         lineStation.splitLineStation(stationPosition, interStation, distanceToSubtract);
     }
+
+    public boolean positionIsNotNone() {
+        return stationPosition != StationPosition.NONE;
```

Comment:
```suggestion
        return stationPosition.isNone();
```

직접 비교보다는 메시지를 전달해볼수있겠네요!

Reply:
객체에 메세지를 보내는 방식에 어느정도 익숙해졌다고 생각했는데 아직 아니었네요 😥  놓친 부분 짚어주셔서 감사합니다!